docs/requirements.md

```diff
@@ -0,0 +1,9 @@
+---
+Order: 10
+Title: Requirements
+Description: Requirements for the Cake.Issues.DupFinder addin.
+---
+The requirements for using the [Cake.Issues.DupFinder addin] are listed in the [release notes] for any specific version.
```

Comment:
Is this OK for you to mention the requirement in release notes?

Reply:
Not sure on what the requirement are except Cake.Issues and the JetBrains commandline tools.

Reply:
Addins are built against a specific API of Cake, in this case also a specific API of Cake.Issues and have requirements for target Framework. See e.g. the [requirements section for Cake.Issues.InspectCode](https://github.com/cake-contrib/Cake.Issues.InspectCode/releases/tag/0.8.0):

```
Requirements
- Cake 0.33.0 or higher
- Cake.Issues 0.8.x
- Targeting .NET Standard 2.0 and .NET Framework 4.6.1
```